setup.py

```diff
@@ -20,5 +35,8 @@
     url='https://github.com/DBuildService/dockerfile-parse',
     license="BSD",
     packages=find_packages(exclude=["tests"]),
+    install_requires=_install_requirements(),
+    setup_requires=['pytest-runner'],
```

Comment:
Can we move this to `tests_requires` so we wouldn't have to modify the spec file?